app/serializers/post_action_type_serializer.rb

```diff
@@ -14,25 +14,25 @@ def is_flag
   end
 
   def name
-    i18n("title")
+    i18n("title", default: object.class.names[object.id])
   end
 
   def description
-    i18n("description", tos_url: tos_url, base_path: Discourse.base_path)
+    i18n("description", vars: { tos_url: tos_url, base_path: Discourse.base_path })
```

Comment:
FYI you can now do this, like JS objects:

```suggestion
    i18n("description", vars: { tos_url:, base_path: Discourse.base_path })
```